R/annotation-data-helper.R

```diff
@@ -20,13 +20,6 @@ getIntronRank <- function(granges) {
   return(intronRank)
 }
 
```

Comment:
We don't need this helper function anymore, which was previously called in `lapply` by `getReducedExonRanges`.